src/renderer/frames/cluster-frame/start-url.injectable.ts

```diff
@@ -4,29 +4,30 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { computed } from "mobx";
-import type { KubeResource } from "../../../common/rbac";
-import isAllowedResourceInjectable from "../../../common/utils/is-allowed-resource.injectable";
 import clusterOverviewRouteInjectable from "../../../common/front-end-routing/routes/cluster/overview/cluster-overview-route.injectable";
 import workloadsOverviewRouteInjectable from "../../../common/front-end-routing/routes/cluster/workloads/overview/workloads-overview-route.injectable";
-import { buildURL } from "../../../common/utils/buildUrl";
 
 const startUrlInjectable = getInjectable({
   id: "start-url",
 
   instantiate: (di) => {
-    const isAllowedResource = (resourceName: string) => di.inject(isAllowedResourceInjectable, resourceName);
-
     const clusterOverviewRoute = di.inject(clusterOverviewRouteInjectable);
     const workloadOverviewRoute = di.inject(workloadsOverviewRouteInjectable);
-    const clusterOverviewUrl = buildURL(clusterOverviewRoute.path);
-    const workloadOverviewUrl = buildURL(workloadOverviewRoute.path);
 
     return computed(() => {
-      const resources: KubeResource[] = ["events", "nodes", "pods"];
+      if (clusterOverviewRoute.isEnabled.get()) {
+        return clusterOverviewRoute.path;
+      }
+
+      if (workloadOverviewRoute.isEnabled.get()) {
+        return workloadOverviewRoute.path;
+      }
 
-      return resources.every((resourceName) => isAllowedResource(resourceName))
-        ? clusterOverviewUrl
-        : workloadOverviewUrl;
+      /**
```

Comment:
I don't follow. Why is this needed?

Reply:
1. Because `workloadOverviewRoute.isEnabled` is typed as `IComputedValue<boolean>` it *could* be `false` so this is a branch of code that _could_ be run.
2. The other option is to just not check `workloadOverviewRoute.isEnabled` at all and then always return `workloadOverviewRoute.path` but then if in the future we change the definition of `workloadOverviewRoute.isEnabled` to not always be defined we might get into the same case again.